app/src/main/res/drawable/ic_splash_simple_logo.xml

```diff
@@ -0,0 +1,16 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
We seem to have a very similar resource already (`ic_logo_large`) that are used during the user registration flows. Could we use that existing resource instead of adding a new one?

Reply:
There is no drawable called `ic_logo_large` . There's one drawable called `ic_icons_logo` which has simple logo and not the text. Zeplin file contains both the logo and the simple text in a single SVG file. Do you want me to verify with Pragati if we should use the text explicitly instead of using it in the SVG file ?

Reply:
Whoops, I meant `logo_large`. There's no `ic_` prefix.

Reply:
Yeah, the logo includes the `Simple` text. That is not localised and we should include it in the image resource.